lib/Auth/Process/Statistics.php

```diff
@@ -27,25 +28,35 @@ public function __construct($config, $reserved)
 
     public function process(&$request)
     {
+        if (empty($this->config->getString('userIdAttribute'))) {
+            if (empty($request['rciamAttributes']['cuid'])) {
+                Logger::warning("[proxystatistics:proccess] There is not any defined User ID. This login will not be saved.");
+                return;
+            } else {
+                $this->userIdAttribute = $request['rciamAttributes']['cuid'];
+            }
+        } else {
+            $this->userIdAttribute = $request['Attributes'][$this->config->getString('userIdAttribute')];
```

Comment:
check if configured uid is not empty; if empty log error and return silently

Reply:
done